### PR TITLE
Users/aedler/installation patch

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

**Review by volllly — CHANGES_REQUESTED** [review]

File: doc/general/installation/installation.md [hunk 1]
```diff
@@ -3,56 +3,138 @@ slug: /poscreators/middleware-doc/general/installation
...
+| ------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | --------------- | --------- | --------------- |
+| `-cashboxid`                   | Sets the CashBoxId. The value is a GUID in format `00000000-0000-0000-0000-000000000000`. | yes                                                          | supported       | supported | supported       |
+| `-accesstoken`                 | Sets the AccessToken for online communication                | yes                                                          | supported       | supported | supported       |
+| `-useoffline`                  | Sets the offline mode. The value is a boolean: true \| false | yes                                                          | supported       | supported | supported       |
+| `-test`                        | Executing as command line program. Basic information is provided in the console. Should be indicated as last parameter, if it is set in connection with others. | no                                                           | supported       | supported | supported       |
```
Comment (volllly) [inline doc/general/installation/installation.md:63]: the test parameter does not need to be the last parameter anymore.

File: doc/general/installation/installation.md [hunk 2]
```diff
@@ -3,56 +3,138 @@ slug: /poscreators/middleware-doc/general/installation
...
+| `test.cmd`                          | Pre-configured command file which executes `fiskaltrust.exe` using the parameter `-test` for starting the fiskaltrust.Middleware as a command line program under Windows, recommended for test and development purpose. | Run the command file `test.cmd` as administrator.<br />For details, see [Test Environment](#test-environment) |                                                              |
+
+### Applying the Cashbox-configuration
+
+During the start of the Online-Launcher (.NET Launcher or Mono-Launcher), the configuration is checked and the configuration-file (usually Configuration-`00000000-0000-0000-0000-000000000000`.json) including the needed packages are downloaded to the fiskaltrust service-folder (usually `C:\ProgramData\fiskaltrust\service`) if necessary. 
+
+**When a new configuration shall be applied, the fiskaltrust service needs to be restarted so that the Launcher checks if a new configuration exists.**
+
+For checking the configuration and downloading the needed packages the Launcher must be able to contact the fiskaltrust packages-server:
```
Comment (TSchmiedlechner) [inline doc/general/installation/installation.md:98]: Maybe add to firewall docs for more complete information as well

File: doc/general/installation/installation.md [hunk 3]
```diff
@@ -3,56 +3,138 @@ slug: /poscreators/middleware-doc/general/installation
...
+| https | TCP      | 443  | packages-sandbox.fiskaltrust.cloud |
+| https | TCP      | 443  | packages.fiskaltrust.cloud         |
+
+### Service folder
+
+The data for the service, including 
+
+- the configuration, 
+- service packages, 
+- the local data storage for receipt data, and
```
Comment (TSchmiedlechner) [inline doc/general/installation/installation.md:113]:
```suggestion
- the local data storage for receipt data if a SQLite database is used, and
```

File: doc/general/installation/installation.md (hunk 1 above)
Comment (aedler-ft) [reply doc/general/installation/installation.md]: cool, thank you a lot for that!

File: doc/general/installation/installation.md [hunk 4]
```diff
@@ -3,56 +3,138 @@ slug: /poscreators/middleware-doc/general/installation
...
+# Installation
 
-### fiskaltrust.Middleware
+For operating the fiskaltrust.Middleware on-premise or off-premise, the components of the fiskaltrust.Middleware need to be configured, downloaded, and installed/started:
 
-#### Windows, Linux, Mac
+## Configuration of Middleware components 
 
-The mentioned platforms use the launcher `fiskaltrust.exe` to execute the fiskaltrust.SecurityMechanism. For Linux and Mac, the launcher can be executed via Mono, version 3.2.8 or higher. For Windows the launcher can be executed via the .NET framework, version 4.0 or higher, or via Mono, version 3.2.8 or higher.
+## (Cashbox configuration)
```
[the remark this answers is not in the document]
Comment (aedler-ft) [reply doc/general/installation/installation.md:12]: tried to rephrase it

File: doc/general/installation/installation.md (hunk 2 above)
Comment (aedler-ft) [reply doc/general/installation/installation.md:98]: I've got the information from there and added it just more explicitly. In the meanwhile, the information has been removed from there. I just found it on Github in the history: https://github.com/fiskaltrust/getting-started/blob/162a76dcd702b3154ee8ee973737f9928b8b1b8d/posdealers/03-sales/firewall-settings.md#daf%C3%BCr-werden-folgende-freigaben-ben%C3%B6tigt

**Review by TSchmiedlechner — APPROVED** [review]

**Pull request merged**